charts/fission-all/values.yaml

```diff
@@ -80,14 +80,25 @@ kafka:
 
 ## Persist data to a persistent volume.
 persistence:
+  ## If true, fission will create/use a Persistent Volume Claim
+  ## If false, use emptyDir
+  ##
   enabled: true
+
+  ## A manually managed Persistent Volume Claim name
+  ## Requires persistence.enabled: true
+  ## If defined, PVC must be created manually before volume will be bound
+  ##
+  # existingClaim:
```

Comment:
It would be better to copy default value to here for reference.

Reply:
It is undefined by default.
if existingClaim is undefined: a PVC named "fission-storage-pvc" will be created.
if existingClaim == "fission-storage-pvc": no PVC will be created. One needs to create a PVC named "fission-storage-pvc" in advance or the chart will fail to install.

The meaning of undefined and set it to "fission-storage-pvc" is different. If it really needs a default value, it will be `null`. (leave it blank also represents null in yaml)

I think it's better to leave it value blank and commented out.
I will add document about default PVC name but I think it will not be this line.

Reply:
You're right, I missed the pvc creation in pvc.yaml.